### PR TITLE
AGP-84: Adding Flock track to AGP-1

### PR DESCRIPTION
**Pull request by LouisGrx**

### CODE REVIEW

**Comment by john-light** [comment]
The time now is after the published deadline of October 10, 2019 at 16:00 UTC for finalizing proposals and moving them to Stage V, and this proposal has not yet been finalized. Per [AGP-1](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-1.md#stage-iv-community-review), this proposal is considered to be withdrawn and this pull request will now be closed:

> If you do not finalize the proposal and request to move the proposal to Stage V before Stage V is scheduled to begin, AGP Editors will consider the proposal withdrawn and close the pull request when Stage V begins.

**Comment by john-light** [comment]
The Aragon Association Board of Directors has voted to push back the ANV-4 schedule by one week. This PR is being re-opened and is included in the Community Review Period which will now last until October 17, 2019 at 16:00 UTC. Please finalize the draft and leave a comment saying "This draft is final and ready to move to Stage V" before the end of the Community Review Period if you would like this proposal to be reviewed by the Aragon Association in Stage V.

Ref: https://forum.aragon.org/t/anv-4-delay-announcement/1367

File: AGPs/AGP-X.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+AGP-1
+
+## Link to proposed change
+
+https://github.com/aragon/AGPs/pull/83
+
+## Motivation for making this change
+
+The Flock application process and the AGP process are currently separate which can create a confusing flow for applicants, AGP editors and the Aragon Association.
```
Comment (john-light) [inline AGPs/AGP-X.md]:
```suggestion
The Flock application process and the AGP process are currently separate which can create a confusing flow for applicants, AGP editors, and the Aragon Association.
```

**Comment by LouisGrx** [comment]
This draft is final and ready to move to Stage V :)

**Pull request merged**